package.json

```diff
@@ -42,6 +43,7 @@
     "walk-sync": "^2.2.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.26.0",
```

Comment:
Required peer dep of `@babel/eslint-parser`.
Forgot to add this when updating the ESLint setup.